src/common/cmake/Common.cmake

```diff
@@ -10,7 +10,12 @@ if(UNIX AND NOT APPLE)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -rdynamic")
 endif()
 
-set(FLATBUFFERS_VERSION "1.7.1")
+# The following is needed because in CentOS, the lib directory is named lib64
+if(EXISTS "/etc/redhat-release" AND CMAKE_SIZEOF_VOID_P EQUAL 8)
```

Comment:
Can we use this to get rid of https://github.com/ray-project/ray/blob/dad465a2bfcd355e8a04afbfa1fd84364051a79e/thirdparty/scripts/build_arrow.sh#L85-L90 and related lines?

Reply:
Relatedly, is this the canonical way of checking for CentOS? https://askubuntu.com/questions/459402/how-to-know-if-the-running-platform-is-ubuntu-or-centos-with-help-of-a-bash-scri suggests `/etc/os-release`.

Reply:
I took it from https://htcondor-wiki.cs.wisc.edu/index.cgi/fileview?f=build/cmake/FindLinuxPlatform.cmake&v=4592599fecc08e5588c4244e2b0ceb7d32363a56, it's officially documented by Red Hat, so should be fine.

Reply:
Getting rid of the copy in arrow will involve changing FindArrow, which is part of arrow.